server/index.js

```diff
@@ -19,12 +19,18 @@ const io = socketIO(server);
 // Pre-Route middlewares
 require("./src/middlewares/pre_route.middleware")(app);
 
+// Express body parser
+app.use(express.urlencoded({ extended: true }));
```

Comment:
register this middleware in the **require("./src/middlewares/pre_route.middleware")(app)** module please